### DISCUSSION TITLE
[Prettier Plugin] Auto-wrap/add line breaks to really long Tailwind classNames

### DISCUSSION THREAD

**Discussion by steven-tey** [tailwindlabs/tailwindcss#9662] [Ideas]
Hey! It would be incredible if this plugin to automatically wrap / add line breaks to really long Tailwind classNames, similar to how prettier automatically breaks named exports of 3 of more functions into multiple lines! :)

**Comment by ony3000** [comment]
Hello! I am leaving a comment because it seems to be a similar topic to #7763.

There is a [prettier plugin](https://www.npmjs.com/package/prettier-plugin-classnames) that I released this year. Maybe this will help.
